gtsfm/utils/io.py

```diff
@@ -438,7 +439,7 @@ def read_points_txt(fpath: str) -> Tuple[Optional[np.ndarray], Optional[np.ndarr
 
 def read_scene(
     images_fpath: str, cameras_fpath: str, points_fpath: str
-) -> Tuple[List[Pose3], List[str], List[Cal3Bundler], np.ndarray, np.ndarray]:
+) -> Tuple[Optional[List[Pose3]], Optional[List[str]], List[Cal3Bundler], Optional[np.ndarray], Optional[np.ndarray]]:
```

Comment:
Why is Cal3Bundler the only thing here which is not optional?

Reply:
No idea